rest_search/views.py

```diff
@@ -23,6 +23,11 @@ def get_schema_operation_parameters(self, view):
 class SearchAPIView(APIView):
     filter_backends = (SearchFilterBackend,)
 
+    def _get_total(self, total):
+        if isinstance(total, dict):
```

Comment:
Let's have an explicit `else` with a comment on each branch explaining which is ES >= 7 and which is ES < 7.